couchbase/metadata.csv

```diff
@@ -8,7 +8,7 @@ couchbase.ram.used,gauge,,byte,,RAM in use,1,couchbase,ram used
 couchbase.ram.total,gauge,,byte,,Total RAM,-1,couchbase,ram total
 couchbase.ram.quota_total,gauge,,byte,,RAM quota,0,couchbase,ram quota
 couchbase.ram.used_by_data,gauge,,byte,,RAM used for data,0,couchbase,ram data used
-couchbase.by_bucket.avg_bg_wait_time,gauge,,second,,Average background wait time,-1,couchbase,avg bg wait
+couchbase.by_bucket.avg_bg_wait_time,gauge,,millisecond,,Average background wait time,-1,couchbase,avg bg wait
```

Comment:
Double checking. Should that be `microsecond` or `milisecond` ? 

Is the description below wrong ?

```
        "avg_bg_wait_time": {
            unit: "microsecond",
            title: "Background Fetch Time",
            desc: "Average background fetch time in microseconds. (measured from avg_bg_wait_time)"
          },
```
from the link provided in desc https://github.com/couchbase/ns_server/blob/926d780026afdf786a2aa42e7aca40e6e5c39c21/priv/public/ui/app/mn_admin/mn_statistics/mn_statistics_description_service.js#L104

Reply:
🤦‍♂ thanks for checking